soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -107,6 +107,21 @@ Randomizer::~Randomizer() {
     this->randomizerMerchantPrices.clear();
 }
 
+std::unordered_map<std::string, RandomizerInf> spoilerFileTrialToEnum = {
+    { "the Forest Trial", RAND_INF_TRIALS_DONE_FOREST_TRIAL },
+    { "l'épreuve de la forêt", RAND_INF_TRIALS_DONE_FOREST_TRIAL },
```

Comment:
Some consistancy between languages :)

l'Épreuve de la Forêt
l'Épreuve du Feu
l'Épreuve de l'Eau
l'Épreuve de l'Esprit
l'Épreuve de l'Ombre
l'Épreuve de la Lumière

Reply:
Issue is, I need those to match what gets put in the spoiler log. I'd have to find where in the 3DS code this is, which I guess I can do, but depending how they do it it might be tricky.

Reply:
Found, changed, and pushed